test/integration.test.js

```diff
@@ -24,12 +25,6 @@ test('integration', async (t) => {
 
   await fastify.ready()
 
-  t.has(
-    fastify.secrets,
-    {
-      test1: 'test content 1',
-      test2: 'test content 2'
-    },
-    'decorates fastify with secret content'
-  )
+  t.assert.deepStrictEqual(fastify.secrets.test1, 'test content 1', 'decorates fastify with secret content')
```

Comment:
why don't we assert in the same way as before?

Reply:
`has` is used to check for a subsect of the object, isn't it?
I think that the node test runner doesn't have this capability yet